### ISSUE TITLE
Broader use for isSimilarTo

### ISSUE THREAD

**Issue by csarven** [schemaorg/schemaorg#582]
`schema:isSimilarTo` is tied (domain/range) to `schema:Product`. Consider relaxing the definition, domain/range, and use `schema:Thing`.

See also `skos:*Match`.

**Comment by mfhepp** [comment]
I hesitate to support this. The meaning of this property is already, by design, pretty ambiguous - broadening its domain and range will make it very difficult to draw meaningful conclusions.

Do you have a use-case for this? Can we expect data to be marked up by this?

**Comment by csarven** [comment]
"is similar to" is the only reasonable way (that I know of) in schema.org which can associate two "things". In a broader sense, it covers `skos:*Match`es. Use case being "comparability".

I'm happy to default to skos (or something else) if I need to use some concepts where the degree of their similarity matters. 

Is there a particular reason why it should be specific to a Product? Is the term "isSimilarTo" intuitive enough such that product is the first thing that comes to mind?

**Comment by mfhepp** [comment]
isSimilar was designed so that related products in product detail pages can be marked up. If you broadened the domain and range, search engines have to first check whether the linked entities are actual products.

If you want SKOS-like features in schema.org, please consider additional properties for this purpose.

**Comment by danbri** [comment]
I could see some value in a reified structure (or pattern using Role) which expressed similarities e.g. amongst CreativeWorks as computed in recommender/ML systems. http://schema.org/isSimilarTo is rather vague already, I'm not allowing arbitrary things to be marked as similar would work. 

Is it true that @csarven isSimilarTo @mfhepp ? Or London isSimilarTo Paris? or the moon? This simplistic construction is overworked even when dealing with product similarities on e-commerce sites, I fear stretching it any further would break it.

**Comment by mfhepp** [comment]
@danbri This is why isSimilarTo is defined as

"This states that a given gr:ProductOrService is similar to another product or service. Of course, this is a subjective statement; when interpreting it, the trust in the origin of the statement should be taken into account."

in

http://www.heppnetz.de/ontologies/goodrelations/v1.html#isSimilarTo

The idea was to have a mechanism in place to publish information on similar items from the perspective of a single shop site. Very little global inferences should be drawn from this, but it can help processing data from the same source (graph), and one could count matching isSimilarTo statements from many sources as a vote + define a threshold.

I think we are in agreement to not broaden the use of this property, even if we may disagree on its value in the current form.

I hope you agree to close this issue, we already have too many others, If you disagree, simply reopen it.

**Comment by danbri** [comment]
We're increasingly encountering issues in which a reasonable vocabulary idea is suggested, but unlikely (unless a more detailed case is made) to be acted upon in forseeable future. Rather than have these pile up I think it is fine to close them, but I have created a new label "Closed but noted vocabulary idea" to avoid the closure of such an issue being thought of as implying that the idea is either rejected or implemented. 

@csarven if you have more use cases, details e.g. of existing sites or related designs do please share them here. Essentially open issues are things that are weighing upon our conscience; closed issues don't necessarily mean "stop talking here!".

**Comment by csarven** [comment]
@mfhepp @danbri Thanks for the feedback. Okay with the issue being closed. Some comments for the future:

While I agree that isSimilarTo can (and will probably) be misused, it is no different than other properties out there which do some sort of an association/mapping, e.g., `skos:exactMatch`. Without additional context (e.g., temporality), even something like an exactMatch is not entirely clear, or subject to mislead.

I also agree that broadening isSimilarTo may not be a good idea, but the immediate alternatives:
1) continue to use a term with an unintuitive description
2) come up with a suite of properties which cover such associations on a granular level

To recap, it was 1 which got me here because I figured it'd be simpler - obviously this is subjective and could even cause more trouble - than doing 2.

**Comment by mfhepp** [comment]
If I remember correctly, there was work that proposed incorporating a subset of SKOS into schema.org. e.g.

https://github.com/schemaorg/schemaorg/issues/251
http://www.w3.org/wiki/WebSchemas/SKOS

I suggest to address your requirement in the miniSKOS work in that issue.
